.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -8,57 +8,68 @@ body:
         Thank you for taking the time to file a bug report. Before creating a new
         issue, please make sure to take a few moments to check the issue tracker
         for existing issues about the bug.
-
-  - type: textarea
-    id: system-info
+  - type: checkboxes
+    id: checks
     attributes:
-      label: System Info
-      description: Please share your system info with us.
-      placeholder: LangChain version, platform, python version, ...
+      label: First Check
+      description: Please confirm and check all the following options.
+      options:
+        - label: I added a very descriptive title here.
+          required: true
+        - label: I used the GitHub search to find a similar question and didn't find it.
+          required: true
+        - label: I searched the LangChain documentation, with the integrated search.
+          required: true
+        - label: I already searched in Google "How to X in LangChain" and didn't find any information.
```

Comment:
Do we care much about this one? If they didn't find it in prev issue or python docs, probably worth an issue to add to python docs?